lib/pages/challenges/challengeCard.dart

```diff
@@ -1,9 +1,8 @@
 import 'package:flutter/material.dart';
+import '../../my_flutter_app_icons.dart';
 
-import 'my_flutter_app_icons.dart';
-
-class ChallengeBallerCard extends StatelessWidget {
-  const ChallengeBallerCard({
+class ChallengeCard extends StatelessWidget {
```

Comment:
Where did the widget that the challengecard returns go?

Reply:
It's right below, maybe github hid the code? You can click the arrows on the left to expand